assets/wix/files.wxs

```diff
@@ -1678,9 +1678,6 @@
               <File Id="filD2DD9066D1A148EBB7AD542DAC529862" KeyPath="yes" Source="$(var.ProductSourcePath)\Modules\PSDesiredStateConfiguration\en-US\about_PSDesiredStateConfiguration.md" />
             </Component>
           </Directory>
-          <Component Id="cmp67C4A6FC5E48487BB31DE411CF38B0B2">
-            <File Id="fil747B2801746E4FFCB93000BBA1089F15" KeyPath="yes" Source="$(var.ProductSourcePath)\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.cat" />
```

Comment:
Revert this change and let the code generate the update and check that in.

Reply:
updated.